src/declaration.h

```diff
@@ -885,4 +885,24 @@ class DeleteDeclaration : public FuncDeclaration
     void accept(Visitor *v) { v->visit(this); }
 };
 
+struct ObjcSelector
```

Comment:
This should go in objc.d, there's no reason to make the fields and methods available throughout the compiler.